src/orion/algo/hyperband.py

```diff
@@ -279,12 +330,14 @@ def register_samples(self, bracket, samples):
                 self.get_id(sample, ignore_fidelity=True, ignore_parent=True)
                 not in self.trial_to_brackets
             ):
+                assert self.brackets is not None
```

Comment:
```suggestion
```
Do we need this? If so it raise a RuntimeError with an informative error message.

Reply:
Fixed in 35ce6867769d77028c87f7bb015adfd349315f85